test/controllers/SubmissionControllerSpec.scala

```diff
@@ -89,49 +89,49 @@ class SubmissionControllerSpec extends SpecBase with AppWithDefaultMockFixtures
         val userAnswers = emptyUserAnswers
         when(mockCacheRepository.get(any(), any())).thenReturn(Future.successful(Some(userAnswers)))
 
-        when(mockApiService.submitDeclaration(any(), any())(any()))
+        when(mockApiService.submitDeclaration(any())(any()))
           .thenReturn(Future.successful(HttpResponse(BAD_REQUEST, "")))
 
         val request = FakeRequest(POST, routes.SubmissionController.post().url)
-          .withHeaders("APIVersion" -> "2.0")
+          .withHeaders("APIVersion" -> "2.1")
```

Comment:
Generally speaking I believe all references to this particular header can be removed